src/utils/action.storage.ts

```diff
@@ -4,12 +4,12 @@
  * Uses $ACTION_PATH if set to store the actions to a user defined location
  * This parses $ARGO_TEMPLATE looking for a `archiveLocation`
  */
-export function getActionLocation(): string | null {
-  if (process.env['ACTION_PATH']) return process.env['ACTION_PATH'];
+export function getActionLocation(): URL | null {
+  if (process.env['ACTION_PATH']) return new URL(process.env['ACTION_PATH']);
```

Comment:
this will fail if "ACTION_PATH" is not a url, eg "/tmp" it is safer to use a URL parser for this.